tools/lsp/server/server.toit

```diff
@@ -140,7 +140,7 @@ class LspServer:
         "toit/report_idle":        (:: report-idle),
         "toit/reset_crash_rate_limit": (:: reset-crash-rate-limit),
         "toit/settings":           (:: settings_.map_),
-        "toit/didOpenMany":        (:: did-open-many it),
+        "toit/analyzeMany":        (:: analyze-many it),
```

Comment:
The naming is pretty weird here. Some of these use underscores, some of the camelCase.

Reply:
Will unify them in a follow-up PR.